sonobuoy-conformance.yaml

```diff
@@ -110,14 +121,17 @@ data:
               fieldPath: spec.nodeName
         - name: RESULTS_DIR
           value: /tmp/results
-        image: gcr.io/heptio-images/sonobuoy:v0.9.0
+        - name: MASTER_URL
+          value: '{{.MasterAddress}}'
+        - name: RESULT_TYPE
+          value: e2e
+        image: gcr.io/heptio-images/sonobuoy:v0.10.0
```

Comment:
so if someone did not submit conformance test against 0.9.0 and this PR got merged, then the people will not able to submit conformance test for Kubenretes 0.9.0?

Reply:
@gyliu513 This change only updates the [version of sonobuoy](https://github.com/heptio/sonobuoy/releases/tag/v0.10.0) that is run for conformance testing, it does not affect the versions of Kubernetes that are being tested for conformance.

Reply:
Got it, thanks @detiber